modules/Bio/EnsEMBL/Production/Pipeline/TaxonomyUpdate/BackUpDatabase.pm

```diff
@@ -0,0 +1,128 @@
+=head1 LICENSE
```

Comment:
We have a module `Bio::EnsEMBL::Production::Pipeline::Common::DatabaseDumper` , can't it be reused in the pipeline instead?